receiver/jaegerreceiver/trace_receiver_test.go

```diff
@@ -137,6 +140,62 @@ func TestGRPCReception(t *testing.T) {
 
 }
 
+func TestGRPCReceptionWithTLS(t *testing.T) {
+	// prepare
+	grpcServerOptions := []grpc.ServerOption{}
+	tlsCreds := receiver.TLSCredentials{
+		CertFile: path.Join(".", "testdata", "certificate.pem"),
+		KeyFile:  path.Join(".", "testdata", "key.pem"),
+	}
+
+	tlsOption, _ := tlsCreds.ToGrpcServerOption()
+
+	grpcServerOptions = append(grpcServerOptions, tlsOption)
+
+	config := &Configuration{
+		CollectorGRPCPort:    14250, // that's the only one used by this test
```

Comment:
Can we allocate this port dynamically to test failure if the port is busy? I believe other tests already do this, we should have a utility function somewhere that returns an available port number.